bundles/org.openhab.binding.dbquery/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,125 @@
+				<default>false</default>
+			</parameter>
+			<parameter name="scalarResult" type="boolean">
+				<label>Scalar result</label>
+				<description>True if the query always return only one single scalar value (only one row and one value-column in this
+					row), otherwise false</description>
+				<default>true</default>
+			</parameter>
+			<parameter name="scalarColumn" type="text" required="false">
+				<label>Scalar column name</label>
+				<description>The column's name that is used to extract scalarResult. If only one column is returned this
+					parameter
+					can be blank</description>
+			</parameter>
+			<parameter name="interval" type="integer" min="0">
+				<label>Interval</label>
+				<description>
+					An interval, in seconds, the query will be repeatedly executed. Default values is 0, which means that
+					query is never executed manually (you need to send ON to execute channel)
+				</description>
+				<default>0</default>
+			</parameter>
+			<parameter name="timeout" type="integer" min="0">
+				<label>Timeout Query</label>
+				<description>
+					A time-out in seconds to wait for the query result, if it's exceeded result will be discarded.
+					Use 0 for
+					no timeout
+				</description>
+				<default>0</default>
+			</parameter>
+
+
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="execute-channel">
+		<item-type>Switch</item-type>
+		<label>Execute Query</label>
+		<description>Send ON to execute the query, the current state tells if the query is running</description>
+	</channel-type>
+	<channel-type id="result-channel-string">
+		<item-type>String</item-type>
+		<label>String Result</label>
+		<description>Execute query and binds result value to channel as a String</description>
+	</channel-type>
+	<channel-type id="result-channel-number">
+		<item-type>Number</item-type>
+		<label>Number Result</label>
+		<description>Execute query and binds result value to channel as a Number</description>
+	</channel-type>
+	<channel-type id="result-channel-datetime">
+		<item-type>DateTime</item-type>
+		<label>DateTime Result</label>
+		<description>Execute query and binds result value to channel as a DateTime</description>
+	</channel-type>
+	<channel-type id="result-channel-contact">
+		<item-type>DateTime</item-type>
+		<label>Contact Result</label>
+		<description>Execute query and binds result value to channel as a Contact</description>
+	</channel-type>
+	<channel-type id="result-channel-switch">
+		<item-type>Switch</item-type>
+		<label>Switch Result</label>
+		<description>Execute query and binds result value to channel as a Switch</description>
+	</channel-type>
+	<channel-type id="parameters-channel">
+		<item-type>String</item-type>
+		<label>JSON Result</label>
+	</channel-type>
+	<channel-type id="correct-channel">
+		<item-type>Switch</item-type>
+		<label>True if last query executed correctly, otherwise false</label>
```

Comment:
```suggestion
		<label>Last Query Worked</label>
		<description>True if last query executed correctly</description>
```
Note the label is short (less then 25 characters) and each word starts with a captial. Description can be longer and give more detail.